website/templates/public/pages/help/files.mako

```diff
@@ -4,9 +4,16 @@
     <p>Each project and component can have its own set of files. This allows you to organize your files into meaningful
         groups like datasets or background research.</p>
     <p>Each file is given a unique URL, meaning that it can be cited or linked to individually.</p>
-    <p>To upload a file, click on "Files" in the grey navigation bar under the Project/Component's title. Here you
-        can drag the file from your desktop on to the screen to upload, or click on the upload button that appears after you've selected a storage add-on in the files pane. To download a file from a project, select the file and then click the Download button that appears in the toolbar; likewise, to delete a file, 
-        select the file and then click the Delete button. You can create folders to organize files by selecting the storage add-on in which you'd like to create a folder and clicking the Create Folder button in the toolbar.</p>
+    <p>To upload a file, click on "Files" in the grey navigation bar under the Project/Component's title. 
+        Drag the file from your desktop onto the screen to upload, or select a storage add-on in the files pane and click the upload
+        button. OSF Storage, the default storage add-on for every project, supports version control. 
+        This means that if you save changes to a file on your computer&mdash;without 
+        changing the name&mdash;you can drag and drop the updated file onto the OSF and a 
+        new version will automatically be created. To view or download old versions, click on the "Revisions" button on the top right
+        of the page when viewing the file.</p>
+    <p>To download a file, select the file and click the "Download" button that appears in the toolbar. To delete a file, 
+        select the file and then click the Delete button. You can create folders to organize files by selecting the 
```

Comment:
We can probably remove "to organize files"; the purpose of folders should be clear to our users.